docs/howto/account-billing/forgot-your-password.md

```diff
@@ -0,0 +1,55 @@
+# Forgot your password or username
+
+Via [{{gui}}](https://{{gui_domain}}) you can reset lost password or reclaim your username.
+
+## Reset lost password
+
+Navigate to <https://{{gui_domain}}> and click _Forgot your password?_.
```

Comment:
Suggestion: "Begin by navigating to the <https://{{gui_domain}}>. At the bottom left-hand side of the page, click on the _Forgot your password?_ button."